storagenode/storagenodedb/pieceinfo.go

```diff
@@ -24,8 +24,8 @@ type pieceinfo struct {
 }
 
 type spaceUsed struct {
-	once sync.Once
 	used int64
+	once sync.Once
```

Comment:
Add a comment why it's in a different order.

Reply:
done